builtin/v12/migration/miner.go

```diff
@@ -67,12 +74,34 @@ func newMinerMigrator(ctx context.Context, store cbor.IpldStore, outCode cid.Cid
 
 	}
 
+	//load sector index from datastore, or create a new one if not found
+	okSectorIndex, prevSectorIndexRoot, err := cache.Read(migration.MarketSectorIndexKey())
```

Comment:
We could consider reading this in `top.go`, and just passing `prevSectorIndexRoot` here so that we don't pass the entire cache here. I don't think it matters too much.